src/components/molecules/EmailPrint.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { EmailPrintButton } from "../atoms/EmailPrintButton";
+
+/**
+ * This component layout the email and print buttons
+ */
+export function EmailPrint(props) {
+  const handleClick = (event) => props.onClick(event.currentTarget.id);
+  return (
+    <div className="md:flex">
```

Comment:
I doubt this component needs to collapse on mobile. It's small enough that its mobile friendly without needing to collapse